src/Error/ExceptionRenderer.php

```diff
@@ -158,13 +158,26 @@ public function render()
             $this->controller->response->header($exception->responseHeader());
         }
         $this->controller->response->statusCode($code);
-        $this->controller->set([
+        $viewVars = [
             'message' => $message,
             'url' => h($url),
             'error' => $exception,
             'code' => $code,
             '_serialize' => ['message', 'url', 'code']
-        ]);
+        ];
+        if ($isDebug) {
+            $rawTrace = $exception->getTrace();
+            $trace = [];
+            foreach ($rawTrace as $line) {
+                if (isset($line['args'])) {
+                    unset($line['args']);
+                }
+                $trace[] = $line;
+            }
+            $viewVars['trace'] = $trace;
```

Comment:
Debugger::trace() already has ways to format backtraces. Perhaps we could refactor that method and have a new method that can format arbitrary stacktrace structures?  `Debugger::formatTrace($exception->getTrace())` for example?